Makefile

```
@@ -139,11 +139,31 @@ dist:
 		--exclude=.vscode \
 		--exclude=.github \
 		--exclude=.gitignore \
-		--exclude=.copr \
 		--transform s/^\./$(PKGNAME)-$(VERSION)/ \
 		. && mv /tmp/$(PKGNAME)-$(VERSION).tar.gz .
 	rm -rf ./vendor
 
+spec: ./dist/srpm/rhc.spec.in
+	sed \
+	    -e 's,[@]SHORTNAME[@],$(SHORTNAME),g' \
+		-e 's,[@]LONGNAME[@],$(LONGNAME),g' \
+		-e 's,[@]VERSION[@],$(VERSION),g' \
+		-e 's,[@]PKGNAME[@],$(PKGNAME),g' \
+		-e 's,[@]TOPICPREFIX[@],$(TOPICPREFIX),g' \
+		-e 's,[@]DATAHOST[@],$(DATAHOST),g' \
+		-e 's,[@]PROVIDER[@],$(PROVIDER),g' \
+		-e 's,[@]PREFIX[@],$(PREFIX),g' \
+		-e 's,[@]BINDIR[@],$(BINDIR),g' \
+		-e 's,[@]SBINDIR[@],$(SBINDIR),g' \
+		-e 's,[@]LIBEXECDIR[@],$(LIBEXECDIR),g' \
+		-e 's,[@]DATAROOTDIR[@],$(DATAROOTDIR),g' \
+		-e 's,[@]DATADIR[@],$(DATADIR),g' \
+		-e 's,[@]SYSCONFDIR[@],$(SYSCONFDIR),g' \
+		-e 's,[@]LOCALSTATEDIR[@],$(LOCALSTATEDIR),g' \
+		-e 's,[@]DOCDIR[@],$(DOCDIR),g' \
+		-e 's,[@]SERVICENAME[@],$(SERVICENAME),g' \
+		./dist/srpm/rhc.spec.in > ./rhc.spec
```

You can then replace these named paths with `make` references:

```suggestion
		$^ > $@
```

I suggest you consider making this a pattern rule. It can turn a single "rhc.spec from rhc.spec.in" rule into a general-purpose "any_file from any_file.in" rule.